src/lesson2/runner.test.ts

```diff
@@ -1,29 +1,39 @@
 import { runner } from "./runner";
 
-test("runner: 1 * 32", () => {
-  expect(runner("1 * 32")).toEqual(32);
-});
+describe("Runner simple cases", () => {
+  it("runner: 1 * 32", () => {
+    expect(runner("1 * 32")).toEqual(32);
+  });
 
-test("runner: 2 * 32", () => {
-  expect(runner("2 * 32")).toEqual(64);
-});
+  it("runner: 2 * 32", () => {
```

Comment:
in case you have `describe` - no need to put that prefix `runner:` manually

when you run `jest` in verbose mode - it renders it as sublists

Reply:
Very good catch, now everything according with the `describe/it` notation!